changelogs/fragments/1221-aws_codebuild-tagging.yml

```diff
@@ -0,0 +1,20 @@
+minor_changes:
+- aws_codebuild - add support for ``purge_tags`` parameter
+  (https://github.com/ansible-collections/community.aws/pull/1221).
+- aws_codebuild - the ``source`` and ``artifacts`` parameters are now optional unless creating a new project
+  (https://github.com/ansible-collections/community.aws/pull/1221).
+- aws_codebuild - add the ``resource_tags`` paramater which takes the dictionary format for tags instead of the list of dictionaries format
```

Comment:
```suggestion
- aws_codebuild - add the ``resource_tags`` parameter which takes the dictionary format for tags instead of the list of dictionaries format
```